Assets/Scripts/RDoor.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class RDoor : MonoBehaviour
```

Comment:
missing the requirecomponents for the assumed below... but this file is a direct copy of the LDoor with just a small change to two variables: limits.min and max which are inversed... Could we just create one base class with a variable for each of these and just set it in inherrited Ldoor and R door?